frontend/app/enrollment/new/components/enrollment-details.tsx

```diff
@@ -16,6 +25,12 @@ export default function EnrollmentDetails() {
                 <div className="font-[500]">Patiënt:</div>
                 <div>{patient ? patientName(patient) : "Onbekend"}</div>
 
+                <div className="font-[500]">E-mailadres:</div>
```

Comment:
I see it's being copied from other rows but `font-medium` is an existing [font weight](https://tailwindcss.com/docs/font-weight) so I don't think there is need to use an arbitrary value.